site/content/en/docs/Reference/API/bases.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "bases"
+linkTitle: "bases"
+type: docs
+weight: 1
+description: >
+    Add resources from a kustomization dir.
+---
+
+{{% pageinfo color="warning" %}}
+The `bases` field was deprecated in v2.1.0. This field will never be removed from the
+kustomize.config.k8s.io/v1beta1 Kustomization API, but it will not be included
+in the kustomize.config.k8s.io/v1 Kustomization API. When Kustomization v1 is available,
+we will announce the deprecation of the v1beta1 version. There will be at least
+two releases between deprecation and removal of Kustomization v1beta1 support from the
+kustomize CLI, and removal itself will happen in a future major version bump.
+
+You can run `kustomize edit fix` to automatically convert `bases` to `resources`.
+{{% /pageinfo %}}
+
+Move entries into the [resources](/docs/reference/api/resources)
+field.  This allows bases - which are still a
+[central concept](/references/kustomize/glossary#base) - to be
```

Comment:
Noting that this link is currently broken. Updating the glossary page should fix this.

Reply:
You are right, but we have 2 choices right now.

1. Make it refer the empty `Bases` page in Concepts section,

```suggestion
[central concept](/docs/concepts/bases) - to be
```

2. or fill up the Glossary section with [this content](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#base) and make it refer to the page

```suggestion
[central concept](/docs/reference/glossary#base) - to be
```

No ones working with either of them, and I can't find out what content is going to be in the Concepts/bases page from this [excel file](https://docs.google.com/spreadsheets/d/1Aaal0Ou10xeDkh_COWkg_r00R0I-Il_t5G2ad6W3jWc/edit#gid=0). So I'm not sure what's the better approach.

cc. @annasong20

Reply:
I went with the first approach because it says "central concept".
https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/a2b60e4bcb498a8e69b48fdb24561b9d09b5df18#diff-a32a54d95c01ac1d058ab31cca31cf654aa970b22b1edc4a92351d257b9e29d3R23

Reply:
Yes, I think this is a great choice!

Documentation in `Concepts` should be more complete than in `Glossary`. Readers of `Glossary` who'd like to learn more would ultimately reference `Concepts`.